### ISSUE TITLE
Non repository object titles missing from BlackLight

### ISSUE THREAD

**Issue by edgartdata** [ycba-cia/blacklight-collections2#288]
@yulgit1 I just noticed today that only the Repository object titles are appearing in BL but they are in the LIDO XML.
An example of that is B1970.1 https://harvester-bl.britishart.yale.edu/oaicatmuseum/OAIHandler?verb=GetRecord&identifier=oai:tms.ycba.yale.edu:107&metadataPrefix=lido which has 1 repository title in TMS, in XML and in BL and 2 alternate titles only in TMS and XML but not in BL:

![Object titles missing from BL are in LIDO](https://user-images.githubusercontent.com/4093644/106919275-ffc47900-66d7-11eb-8b73-f94287f86839.png)

![Object titles missing from BL](https://user-images.githubusercontent.com/4093644/106919039-c1c75500-66d7-11eb-8fdb-9ebbd214d9c4.png)

**Comment by yulgit1** [comment]
Those are the current mapping rules we have, the "Title" label in BL maps to the lido:type="Repository title" where also lido:pref="preferred".  Are you asking to display other titles in BL?

Interesting you chose this example, Ed is reexamining the current accepted identities of Tunstal and William Cavendish.  Perhaps that's why you're looking at this now?

**Comment by edgartdata** [comment]
We have records where it is important to have multiple titles because they belong to a set for example and we used to have multiple object titles certainly in Vufind and I also believe in BL. We control which of these multiple titles get pushed to BL from TMS. Can we review the mapping rules soon?

As for my choice of B1970.1 I did choose it because a number of us are working on it now and are familiar with its various titles. Ed and I have talked about his research and I am happy to update the repository/primary/preferred title as soon as I get the green light. I would be in favor of publishing to BL the soon to be former repository title as second/alternate title in addition to the new title that Ed will decide on as a way to emphasize the research he is doing and give a sense/be transparent about the past description of the work of art.
